cdc/kv/client.go

```diff
@@ -782,12 +819,8 @@ func (s *eventFeedSession) requestRegionToStore(
 				continue
 			}
 
-			// Wait for a while and retry sending the request
-			time.Sleep(time.Millisecond * time.Duration(rand.Intn(100)))
```

Comment:
do not need sleep in the new change ?

Reply:
Yes, it will be rate-limited in `onRegionFail`.